homeassistant/components/manual_mqtt/alarm_control_panel.py

```diff
@@ -15,16 +16,17 @@
     SUPPORT_ALARM_TRIGGER,
 )
 from homeassistant.const import (
+    CONF_ARMING_TIME,
     CONF_CODE,
     CONF_DELAY_TIME,
     CONF_DISARM_AFTER_TRIGGER,
     CONF_NAME,
-    CONF_PENDING_TIME,
     CONF_PLATFORM,
     CONF_TRIGGER_TIME,
     STATE_ALARM_ARMED_AWAY,
     STATE_ALARM_ARMED_HOME,
     STATE_ALARM_ARMED_NIGHT,
+    STATE_ALARM_ARMING,
     STATE_ALARM_DISARMED,
     STATE_ALARM_PENDING,
     STATE_ALARM_TRIGGERED,
```

Comment:
We missing disarming?  I see you added "arming" but there should also be a disarming state no?

Reply:
No, because there can never be a delay from when the "disarm" command is issued to when the system goes into the disarmed state.  So it always goes from an arming, armed, pending, or triggered state directly to disarmed.

Reply:
Right, I was thinking about how the MQTT Alarm works, which is actually the reverse, its a hardware device and therefore may have a disarm delay.  However, the Manual MQTT Alarm is server-side and would be instant.  Thanks for the clarification!